testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveBucketedTables.java

```diff
@@ -340,16 +346,12 @@ public void testBucketingWithUnsupportedDataTypes(BucketingType bucketingType, S
                                     "   n_union ROW(tag tinyint, field0 integer, field1 varchar),\n" +
                                     "   n_struct ROW(field1 integer, field2 varchar)\n" +
                                     ")\n" +
-                                    "WITH (\n" +
-                                    "   bucket_count = 2,\n" +
-                                    "   bucketed_by = ARRAY['%s'],\n" +
-                                    "   bucketing_version = %s,\n" +
-                                    "   format = 'ORC',\n" +
-                                    "   sorted_by = ARRAY[]\n" +
-                                    ")",
+                                    "WITH (\\E.*" +
+                                    "bucket_count = 2.*" +
+                                    "bucketed_by = ARRAY\\['%s'\\].*bucketing_version = %s.*",
                             tableName,
                             columnToBeBucketed,
-                            getExpectedBucketVersion(bucketingType))));
+                            getExpectedBucketVersion(bucketingType)), Pattern.DOTALL));
```

Comment:
it would be safer to apply DOTALL more selectively, if possible, like `(?s:.*)`

Reply:
Hmm. I think I would wrap whole pattern anyway. Am I missing sth?